### ISSUE TITLE
Export export_resource_package as folder

### ISSUE THREAD

**Issue by oeway** [bioimage-io/core-bioimage-io-python#269]
It seems when calling `export_resource_package` it always create a zip package but it takes significant more time to load the zip package for inference compared to an extracted directory. It would be nice to allow by passing the package creation and only export it as a folder.

**Comment by constantinpape** [comment]
Good point. I don't know much about the internals though, so @FynnBe would need to have a look at this.
In the meantime you could just unzip afterwards (easy to automate via `zipfile`).
